src/VuetifyFileManager.vue

```diff
@@ -88,14 +86,17 @@ export default {
     },
     methods: {
         folderSelected(selectedFolderId) {
-            this.getFolderContent(selectedFolderId).then((content) => {
+            this.selectedFolder = selectedFolderId;
+
+            this.getFolderContent(this.selectedFolder).then((content) => {
                 this.files = content;
             });
         },
         callGetFolders() {
             this.getFolders().then(folders => {
                 this.folders = folders;
-                this.folderSelected(folders[0].id);
+                this.selectedFolder = this.selectedFolder === null ? folders[0].id : this.selectedFolder;
```

Comment:
And then so, a variable of type `Number` should never switch between a number and `null`, only objects should.

If it would be a case for an object, `this.selectedFolder || folders[0].id` would be the shorthand of doing this.